src/org/igniterealtime/xiff/data/archive/ChatStanza.as

```diff
@@ -0,0 +1,78 @@
+/**
+ * Created by kvint on 06.11.14.
+ */
+package org.igniterealtime.xiff.data.archive {
```

Comment:
Please follow the same code style as the existing files has, braces at the next line